src/constants/urls.js

```diff
@@ -0,0 +1,9 @@
+const url = {
+  index: '/',
+  signUp: '/sign_up',
+  signIn: '/sign_in',
+  cart: '/cart',
+  productDetail: '/:productId',
```

Comment:
This parametrized route should be function 
```js
productId => `/${productId}`
```
so when you will want to change route to any product detail, you'll just call urls.productDetail(1) for product with id 1

Reply:
DONE!